### PR TITLE
Use onclick handler from parameter passed from component/controller property

### PR DESCRIPTION
**Pull request by nskrypnik**
# MINOR

### CODE REVIEW

**Review by psbanka — APPROVED** [review]
👍

**Comment by sglanzer-fire** [comment]
What does this cover?  From the comment it appears to be something like

```
{{frost-button
  onClick=(action (mut foo) 'bar'))
}}
```

Can you confirm?

**Comment by sglanzer-fire** [comment]
@nskrypnik @psbanka ^ can either of you clarify?

File: addon/components/frost-button.js [hunk 1]
```diff
@@ -210,15 +210,28 @@ export default Component.extend(PropTypeMixin, {
...
   onclick: Ember.on('click', function (event) {
     if (!ViewUtils.isSimpleClick(event)) {
       return true
     }
 
-    if (!this.get('disabled') && typeOf(this.attrs.onClick) === 'function') {
-      this.attrs.onClick(this.get('id'))
+    const onClickHandler = this._getOnClickHandler()
+    if (onClickHandler && !this.get('disabled')) {
+      onClickHandler(this.get('id'))
```
Comment (sglanzer-fire) [inline addon/components/frost-button.js:234]: Why id?

**Pull request merged**